packages/flutter_test/lib/src/binding.dart

```diff
@@ -395,6 +395,18 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
     assert(debugAssertNoTransientCallbacks(
       'An animation is still running even after the widget tree was disposed.'
     ));
+    assert(() {
```

Comment:
we've moved to throwing a FlutterError rather than having a string in the assert